api/openapi-spec/swagger.json

```diff
@@ -218,6 +218,10 @@
           "description": "Name of template to execute",
           "type": "string"
         },
+        "templateRef": {
```

Comment:
Are you sure this belongs to the `Template` struct? AFAIK it should be in the `DagTask` as alongside the `template` attribute.